api/monitor/types.go

```diff
@@ -5,8 +5,7 @@ import (
 	"github.com/go-redis/redis"
 )
 
-type BatchListener struct {
-	Rdb   *redis.Client
-	Admin *kafka.AdminClient
-	c     *kafka.Consumer
+type BatchController struct {
+	Rdb        *redis.Client
+	KafkaAdmin *kafka.AdminClient
```

Comment:
these should not be exposed, I think you should only expose methods on BatchController

Reply:
Done